pyproject.toml

```
@@ -33,48 +36,39 @@ classifiers = [
     "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
     "Natural Language :: English",
     "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
     "Topic :: Scientific/Engineering :: Bio-Informatics",
     "Topic :: Scientific/Engineering :: Information Analysis",
     "Topic :: Scientific/Engineering :: Visualization",
     "Topic :: Scientific/Engineering",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: Utilities",
-    # "License :: OSI Approved :: BSD License",
-    # "Typing :: Typed",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Typing :: Typed",
 ]
 # add your package dependencies here
-dependencies = [
-    "napari>=0.4.13",
-    "omero-py",
-    "omero-rois",
-    "importlib-metadata; python_version < '3.8'",
-]
+dependencies = ["napari>=0.4.13", "omero-py", "omero-rois"]
```

```suggestion
dependencies = ["napari>=0.4.13", "omero-py", "omero-rois", "qtpy"]
```
Given the imports from qtpy, I think we should depend on it, even tho napari does cover it?

---

Oh and dask.array too i guess